src/Core/tests/DeviceTests/Handlers/ProgressBar/ProgressBarTests.Android.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Maui.Handlers;
+using AProgressBar = Android.Widget.ProgressBar;
+
+namespace Microsoft.Maui.DeviceTests
+{
+	public partial class ProgressBarHandlerTests
+	{
+		AProgressBar GetNativeProgressBar(ProgressBarHandler progressBarHandler) =>
+			(AProgressBar)progressBarHandler.View;
+
+		double GetNativeProgress(ProgressBarHandler progressBarHandler) =>
+			(double)GetNativeProgressBar(progressBarHandler).Progress / ProgressBar.Maximum;
+
+		Task ValidateNativeProgressColor(IProgress progressBar, Color color, Action action = null) =>
+		   ValidateHasColor(progressBar, color, action);
+
+		Task ValidateHasColor(IProgress progressBar, Color color, Action action = null)
+		{
+			return InvokeOnMainThreadAsync(() =>
+			{
+				var nativeProgressBar = GetNativeProgressBar(CreateHandler(progressBar));
+				action?.Invoke();
+				nativeProgressBar.AssertContainsColor(color);
```

Comment:
I probably did a bad thing and made this async without adding "Async" to the name. This needs to return the value so the test can run correctly.

```suggestion
				return nativeProgressBar.AssertContainsColor(color);
```